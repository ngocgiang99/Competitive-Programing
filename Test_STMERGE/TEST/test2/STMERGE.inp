10
2 8
557740213 781982137 555607330 200689406 565815880 356484285 325418731 670345355 
854130960 518292096 373711491 243137466 229609809 974764248 548970062 216796799 
7 3
344830195 336418065 477337235 
960992728 929171284 416239075 
767986013 564677483 920878233 
933286316 158120757 302003558 
949879845 820553055 489821087 
377056991 56020382 951646258 
326683744 595645226 919235816 
2 1
250645980 
68911657 
2 3
971620043 145410502 97802045 
293417503 26276363 653279331 
8 5
609702386 676073702 127406319 804052654 212777235 
667146867 132405723 619713187 239139398 730821233 
566584153 426669138 720041630 403917442 412958367 
889041792 695405936 970531470 785300259 851664434 
221720657 197790392 443835544 815912110 215240471 
224959846 417616116 657274743 963963862 720521020 
481042540 909316948 405557476 224804128 828914036 
996177224 892958933 969540663 381976771 829376154 
4 1
242712908 
674064078 
780756725 
699263952 
8 5
614577551 294456797 244352292 275136817 390327263 
944998960 267824028 457340329 324356028 315333423 
947776931 582358936 945782587 8489342 680424302 
852216258 107404463 622683140 282147186 376203563 
511388848 888880099 582874724 885131034 58365968 
608803971 282985273 741359380 85748198 308726151 
215737125 92289756 673674630 96661717 163063199 
744005240 536245382 592583313 114985562 601548846 
4 9
515450700 950046850 920191032 76116778 28574890 189442104 344028500 33232272 783755497 
519629911 272613027 374992761 593911998 166564899 679080187 236065866 641052446 992486178 
292957040 577971623 105917245 371804439 557605207 722846582 856261778 915139593 267392582 
497023963 559285456 781576763 114167825 968713020 926772206 570231348 923154638 406368830 
9 8
597389521 472093125 978857433 338686668 51282608 509763320 368739793 547126710 
425413438 917302585 101853948 403725561 598935955 968505392 766959247 544467548 
628584705 858990528 502138878 661956066 181402399 407327246 21236992 411444178 
554617282 32980352 93231497 47071920 709976317 993790139 63206861 24145217 
348821800 870781604 778975810 664727178 434152661 614382687 457572253 931813329 
98176731 209221458 938550483 996010137 727934542 570139937 408303897 792326313 
175779460 138327986 1635300 633331205 345522961 508111658 423208681 554608370 
709995979 891262958 150560284 636193781 986726598 608880290 720448436 563520467 
673170519 936373803 319776804 28103569 232616212 736929712 230964770 296509557 
1 4
318356131 323410435 586020771 356355708 
