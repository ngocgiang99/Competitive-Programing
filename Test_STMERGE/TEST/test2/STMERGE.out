430299215
326683744
68911657
97802045
212777235
242712908
359337854
439601102
127915600
318356131
